src/apiMocks/mockData.ts

```diff
@@ -0,0 +1,202 @@
+      data_broker_id: 6245106222891008,
+    },
+    {
+      id: 2422458204291072,
+      age: 8783681059553280,
+      url: "https://raw-implication.name",
+      link: "https://utilized-patina.biz/",
+      emails: [
+        "Austin_Ratke20@example.com",
+        "Conrad_Dibbert@example.net",
+        "Roberta93@example.org",
+      ],
+      phones: [
+        "545-229-2095 x433",
+        "(640) 265-5486 x461",
+        "270.888.9724 x60324",
+      ],
+      status: "new",
+      scan_id: 3231762513657856,
+      addresses: [
+        {
+          zip: "38895-6870",
+          city: "West Edwardofort",
+          state: "Louisiana",
+          street: "Huel Islands",
+        },
+        {
+          zip: "71651",
+          city: "Fort Kiarraburgh",
+          state: "New Jersey",
+          street: "Grady Overpass",
+        },
+        {
+          zip: "19071-2650",
+          city: "North Violetberg",
+          state: "Georgia",
+          street: "Karson Greens",
+        },
+      ],
+      last_name: "Jenkins",
+      relatives: ["Anthony Spinka", "Roxanne Pagac MD", "Sheri Gerlach"],
+      created_at: "2022-12-20T19:56:25.278Z",
+      first_name: "Augusta",
+      profile_id: 8939173994037248,
+      updated_at: "2023-05-08T12:05:54.004Z",
+      data_broker: "fragrant-abbreviation.biz",
+      middle_name: "Logan",
+      data_broker_id: 3216214182592512,
+    },
+    {
+      id: 5430061092044800,
+      age: 287951200190464,
+      url: "https://chubby-breakthrough.org",
+      link: "https://outlandish-nerve.biz/",
+      emails: [
+        "Alyce.Roberts16@example.org",
+        "Hiram_Skiles@example.net",
+        "Bruce32@example.net",
+      ],
+      phones: ["1-402-293-7617 x15609", "593-714-2275 x290", "1-224-949-6071"],
+      status: "new",
+      scan_id: 6575382260088832,
+      addresses: [
+        {
+          zip: "54177-5937",
+          city: "Blandatown",
+          state: "Minnesota",
+          street: "Schiller Points",
+        },
+        {
+          zip: "29714",
+          city: "Barnstable Town",
+          state: "West Virginia",
+          street: "Effertz Pike",
+        },
+        {
+          zip: "78097-9935",
+          city: "Whitefield",
+          state: "Maryland",
+          street: "Heidenreich Circles",
+        },
+      ],
+      last_name: "Okuneva",
+      relatives: ["Elias Dickens", "Bobby Durgan", "Mamie Pouros DDS"],
+      created_at: "2022-09-15T15:19:51.973Z",
+      first_name: "Ursula",
+      profile_id: 6133019091927040,
+      updated_at: "2022-09-22T15:23:08.758Z",
+      data_broker: "long-term-try.net",
+      middle_name: "Addison",
+      data_broker_id: 1875589522784256,
+    },
+  ],
+};
+
+/**
+ * Generates scan result with randomly-generated mock data.
+ *
+ * @returns {OneRepScanResult} - A single scan result.
+ */
+export function createRandomScan(): OneRepScanResult {
+  return {
+    id: faker.number.int() || null,
+    age: faker.number.int(),
```

Comment:
I left a confusing, rambling message above, but if this is meant to be a human age, we might want to cap this at something more reasonable (like 100 or 105).

https://fakerjs.dev/api/number.html#int
```js
age: faker.number.int(100),
age: faker.number.int({ max: 105 }),
age: faker.number.int({ min: 14, max: 97 });
```